### DISCUSSION TITLE
both the same size

### DISCUSSION THREAD

**Discussion by jamesblunt** [upscayl/upscayl#520] [Q&A]
### Checklist

- [X] I have checked that this issue isn't addressed in [the FAQ](https://github.com/upscayl/upscayl#-faq).
- [X] I have checked that this issue isn't addressed in any other issue or pull request.

### Description

Why after upgrading to the latest version, although the size of the original pictures is similar, the size of the pictures after the quality improvement is exactly the same

**Comment by NayamAmarshe** [comment]
What do you mean? Could you please explain?

**Reply by jamesblunt** [reply]
I used batch processing mode to process ten pictures. The sizes of these ten pictures were different, but the processed results were all the same size.

**Reply by NayamAmarshe** [reply]
Did you notice any quality loss?

**Comment by aaronliu0130** [comment]
Possibly related to #516
